plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryWithProxyConnectorSmokeTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.bigquery;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.io.Resources;
+import io.trino.testing.QueryRunner;
+import io.trino.testing.containers.MitmProxy;
+
+import java.io.File;
+import java.net.URISyntaxException;
+
+public class TestBigQueryWithProxyConnectorSmokeTest
```

Comment:
do we really need full connector test? just a few variation of queries:

- storage API (grpc)
- insert api
- query api
- information_schema/listing

That would cover both GRPC and HTTP. smoke tests also feel overkill.